packages/core/src/index.html

```diff
@@ -161,5 +169,97 @@ <h3>Disabled</h3>
         console.log('wayBlur');
       });
     </script>
+
+    <hr>
+    
+    <section>
+      <script>
+        const selectOptions = [
+            { label: 'Arthur Benjamin Dickinson', value: 1001, age: 32 },
+            { label: 'Alexa van der Borre', value: 301, age: 29 },
+            { label: 'Benny Balotelli', value: 2 },
+            { label: 'Eddy Masters', value: 451 },
+            { label: 'Dennis Lei', value: 123 },
+            { label: 'Lynn Kwan', value: 4 },
+            { label: 'Mai Nguyễn', value: 39 },
+            { label: 'Michael Sigþórsson', value: 35 },
+          ];
+      </script>
+      <h2>way-autosuggest</h2>
+      <section>
+        <h3>Basic usage</h3>
+        <way-autosuggest name="autosuggest" value='[{ "label": "Benny Balotelli", "value": 2 }]'></way-autosuggest>
+        <div>
+          <p>autosuggest data:</p>
+          <code class="code-autosuggest-basic"></code>
+          <p>hidden input data:</p>
+          <code class="code-autosuggest-basic-hidden"></code>
+        </div>
+        <script>
+          (function(){
+            const autosuggest = document.querySelector('way-autosuggest');
+    
+            autosuggest.options = selectOptions;
+            autosuggest.config = {
+              multi: true,
+              maxTags: 6,
+              selectedText: 'geselecteerd',
+              tagColor: 'rgba(60, 170, 180, 1)',
```

Comment:
**issue**: In terms of customizability I would like to use a CSS var here, for example --way-color-primary, --way-color-secondary, or introduce a --way-color-tertiary. Or one of the shades/tints of those colors.

Reply:
- [ ] remove tagColor setting and change it to a css var